tests/test-rgw.yml

```diff
@@ -11,6 +11,8 @@
       delay: 2
     - name: Run a swift stat
       shell: ". /root/openrc && swift stat"
+    - name: Upload a file using swift cli
+      shell: ". /root/openrc && swift upload test_cont /etc/hosts --object-name hosts"
```

Comment:
What about the os_object module here?  We can replace the import of the openrc with a clouds.yml and set the cloud = to the test cloud auth.